### DISCUSSION TITLE
Welcome to Mirth Connect Discussions!

### DISCUSSION THREAD

**Discussion by brentmoen** [nextgenhealthcare/connect#4482] [General]
## 👋 Welcome!

**We request that you add your Company name and website to your GitHub profile** before participating in this community, thanks!

We’re using Discussions as a place to connect with other members of the Mirth community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

Please take a moment to briefly introduce yourself and tell us how you use Mirth Connect!

**Comment by usharaniyerra** [comment]
Hi @brentmoen 

Myself Usha Rani. We are using Mirth connect in our organisation. We have one issue while updating the mirth self-signed certificate to a certfifcate from our CA.

After updating the certificate, the login console from the browser with the url https://DNS:8443 showing invalid credentials. But if we revert the cert with self-signed we are able to properly logging in.

Could you please help us whether we are missing any step when updating the self-signed certs with our company's certs?

**Comment by siddharthsharma9258** [comment]
Hi,

My name is Siddharth (you can call me Sid :). I used to work extensively on Mirth 5 years ago and was an active member on the forums. I don't work on Integrations but still I like to help people sort out the issues they are facing in Mirth or Healthcare workflows in general. Currently situated in Canada.

Mirth landscape has changed over the years - forums got a new look, a new discussion board under Github along with Slack. Just yesterday I saw the latest 4.1.1 - cool new features in it. Things are looking pretty great!

Happy to help!

**Comment by usharaniyerra** [comment]
Hi Sid,

Thanks for your reply! Can you please help me out with my issue mentioned in the above comment.

**Comment by Madhava-Marri** [comment]
Hello Team,
I am using Nextgen for my application, i am able to get the https://nextgen-connect.teleteachers.us/webadmin/Index.action for 8080.
but i am trying to click on ACCESS SECURE SITE in the page, i am unable to load the page, any inputs on this.

I am running my Nextgen application in EKS cluster.

Screenshot 2025-01-23 114222
image

Please help, any feed back is valuable.
thank you

**Comment by robertogani** [comment]
Hello,

is there a smart method to convert XML (HL7) like this


> 
.....
             <EVN.5>
                   <XCN.1>DNTLBT93A47C415F</XCN.1>
                    <XCN.2>
                        <FN.1 />
                    </XCN.2>
                    <XCN.3 />
                    <XCN.9>
                        <HD.1 />
                    </XCN.9>
                    <XCN.16>
                        <CE.1></CE.1>
                        <CE.3></CE.3>
                    </XCN.16>
                    <XCN.22>
                        <CWE.2></CWE.2>
                        <CWE.3></CWE.3>
                        <CWE.5></CWE.5>
                        <CWE.6></CWE.6>
                        <CWE.8></CWE.8>
                        <CWE.9></CWE.9>
                    </XCN.22>
                </EVN.5>
......
> 

to a pipe form HL7 v2?

Mirth use a positional form and his default conversion does not work.

**Reply by jonbartels** [reply]
There is but you will not get an answer here. Make a new discussion post, use the GitHub formatting tools to format your XML.

**Reply by robertogani** [reply]
I already solved it my self. It is necessary check true on useStrictParser and useStrictValidation properties.
Thanks
